src/pages/RemoveLiquidity/index.tsx

```diff
@@ -625,9 +627,16 @@ export default function RemoveLiquidity({
             )}
             <div style={{ position: 'relative' }}>
               {!account ? (
-                <ButtonLight onClick={toggleWalletModal}>
-                  <Trans>Connect Wallet</Trans>
-                </ButtonLight>
+                <TraceEvent
+                  events={[Event.onClick]}
+                  name={EventName.CONNECT_WALLET_BUTTON_CLICKED}
+                  properties={{ received_swap_quote: false }}
```

Comment:
i don' think you need to have `received_swap_quote` as false everywhere, could it just be undefined? no strong opinion though

Reply:
i prefer being explicit here too, even in cases like here where it is not possible for received_swap_quote to ever be true.